test/azul_test_case.py

```diff
@@ -364,6 +365,22 @@ def _patch_lambda_env(cls):
         cls.addClassPatch(patch.dict(os.environ,
                                      AWS_LAMBDA_FUNCTION_NAME='unit-tests'))
 
+    valid_schema_domains = [
+        'schema.humancellatlas.org',
+        'schema.dev.data.humancellatlas.org',
+        'schema.staging.data.humancellatlas.org',
+        'schema.integration.data.humancellatlas.org',
+    ]
+
+    @classmethod
+    def _patch_valid_schema_domains(cls):
+        from humancellatlas.data.metadata import (
+            api,
+        )
```

Comment:
What's the reason for making this a local import?

Reply:
I believe we added this as a local import during our investigation in PL how validation cold be added to `links` and `descriptor`. I might of missed that I was supposed un-localize the import when cleaning up our rough draft.
I've added a fixup commit to get rid of the local import.

Reply:
> I've added a fixup commit to get rid of the local import.

Local import is still there.